src/ebay-section-title/cta.tsx

```diff
@@ -1,37 +1,25 @@
 import React, { ComponentProps, FC, ReactNode } from 'react'
 import classNames from 'classnames'
-import { EbayIcon, Icon } from '../ebay-icon'
 
-type Props = ComponentProps<'div'> & {
+type Props = ComponentProps<'a'> & {
   ctaText?: ReactNode;
```

Comment:
`  ctaText: ReactNode;`  
  based on the logic below

Reply:
On section-title ctaText is not required, since we are already handling the empty case, don't need to make it required